### DISCUSSION TITLE
现在的武器抽卡图是个大问题

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#937] [General]
没有稳定的来源

**Comment by mark9804** [comment]
测试服的包体可追踪了，之后会更难

**Reply by Arondight** [reply]
我想把抽卡改回内鬼网，现在3.1的武器抽卡图还没有呢

**Reply by mark9804** [reply]
现在是3.几啊，这半个月都找黑奴代打的

**Reply by BTMuli** [reply]
这个版本是3.1

**Reply by Arondight** [reply]
明天就做

**Reply by BTMuli** [reply]
只有 3.1 新出的武器 `gacha` 图缺失是吗

**Reply by Arondight** [reply]
是的，明天我改一下智障爬虫就行，不用手工做

**Reply by BTMuli** [reply]
[`Project Amber`](ambr.top) 没有 `gacha` 图的资源，可以爬 [`这个`](https://genshin.honeyhunterworld.com/new-in-3-1/?lang=CHS)

**Reply by mark9804** [reply]
这个就是我们说的内鬼网，因为会裁切图片所以不到万不得已我不是很想用

~~现在是万不得已~~

> [`Project Amber`](ambr.top) 没有 `gacha` 图的资源，可以爬 [`这个`](https://genshin.honeyhunterworld.com/new-in-3-1/?lang=CHS)

**Comment by Arondight** [comment]
https://github.com/Arondight/Adachi-BOT/commit/320b7b9a538fde6e46492b83bc859d93772fee36

@Mark9804 简单处理了一下，老大哥看着没啥问题就同步一下 master 吧

**Comment by mark9804** [comment]
老大哥看着没问题就推上去吧，这个属于小问题了，武器没人会在意拉伸的
